Deploy/examples/python_agent/python_agent/app.py

```diff
@@ -1,9 +1,13 @@
 from flask import Flask, jsonify, request
 from .poly_model import PolyModel
+from twautils.log import twa_logging
 
 # Create the Flask app object
 app = Flask(__name__)
 
+# Initialise logger
+logger = twa_logging.get_logger("dev")
```

Comment:
With the changes to log/\_\_init\_\_.py, this could be either
```suggestion
from twautils.log import get_logger
# Create the Flask app object
app = Flask(__name__)
# Initialise logger
logger = get_logger("dev")

```
or:
```suggestion
import twautils.log
# Create the Flask app object
app = Flask(__name__)
# Initialise logger
logger = twautils.log.get_logger("dev")

```

Reply:
Updated using second option.